src/box/xrow.c

```diff
@@ -2054,14 +2058,16 @@ struct replication_request {
 	struct tt_uuid *instance_uuid;
 	/** IPROTO_INSTANCE_NAME. */
 	char *instance_name;
-	/** IPROTO_VCLOCK. */
+	/** IPROTO_VCLOCK or IPROTO_CHECKPOINT_CURSOR.VCLOCK. */
 	struct vclock *vclock;
```

Comment:
Nit: Better add a new field to the struct for `checkpoint_vclock`.

Reply:
Done